RUFAS/routines/manure/manure_treatments/manure_treatment_daily_output.py

```diff
@@ -156,18 +156,23 @@ class ManureTreatmentDailyOutput(LiquidManurePortionProtocol):
         Daily mass of solid manure, kg.
     solid_manure_daily_mass_unit: MeasurementUnits
         Unit for solid_manure_daily_mass.
-    biogas: float
-        Amount of biogas produced, m^3.
-    biogas_unit: MeasurementUnits
-        Unit for biogas.
-    biogas_energy_content: float
-        Energy content of biogas, MJ/m^3.
-    biogas_energy_content_unit: MeasurementUnits
-        Unit for biogas_energy_content.
+    methane_generation_mass : float, default 0.0
+        Mass of methane produced from anaerobic digestion, kg.
+    methane_generation_mass_unit: MeasurementUnits, default MeasurementUnits.KILOGRAMS_PER_DAY
+        Unit for methane_generation_mass.
+    methane_energy_content : float, default 0.0
+        Energy content of methane, MJ.
+    methane_energy_content_unit: MeasurementUnits, default MeasurementUnits.MEGAJOULES
+        Unit for methane_energy_content.
```

Comment:
```suggestion
   biogas: float, default 0.0
        Mass of methane produced from anaerobic digestion, kg.
   biogas_unit: MeasurementUnits, default MeasurementUnits.KILOGRAMS_PER_DAY
        Unit for methane_generation_mass.
    biogas_energy_content : float, default 0.0
        Energy content of methane, MJ.
   biogas_energy_content_unit: MeasurementUnits, default MeasurementUnits.MEGAJOULES
        Unit for methane_energy_content.
```
Reverting output name changes to maintain backwards compatibility